src/bundle/ui-dev/src/modules/multi-file-upload/services/multi.file.upload.service.js

```diff
@@ -140,38 +164,81 @@ const prepareStruct = ({ parentInfo, config, languageCode }, data) => {
 
     return getContentTypeByIdentifier(config, mapping.contentTypeIdentifier)
         .then((response) => response.json())
-        .catch(() => window.ibexa.helpers.notification.showErrorNotification('Cannot get content type by identifier'))
+        .catch(() =>
+            window.ibexa.helpers.notification.showErrorNotification(
+                Translator.trans(
```

Comment:
You can add `const { Translator } = window;` at the top of the file to eliminate eslint warning.